cmd/limits.go

```diff
@@ -0,0 +1,117 @@
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/exoscale/egoscale"
+	"github.com/spf13/cobra"
+)
+
+type LimitsItemOutput struct {
+	Resource string `json:"resource"`
+	Used     int    `json:"used"`
+	Max      int    `json:"max"`
+}
+
+type LimitsOutput []LimitsItemOutput
+
+func (o *LimitsOutput) toJSON()  { outputJSON(o) }
+func (o *LimitsOutput) toText()  { outputText(o) }
+func (o *LimitsOutput) toTable() { outputTable(o) }
+
+var limitsCmd = &cobra.Command{
+	Use:   "limits",
+	Short: "Current account limits",
+	Long: fmt.Sprintf(`This command lists the safety limits currently enforced on your account.
```

Comment:
s/enforced/attached/ ?

Reply:
I referred to the description of the Exoscale console

Reply:
ok